CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 ## 0.6.1 (Unreleased)
 
 IMPROVEMENTS:
- * core: Add autocomplete functionality for flags for all cli commands
+ * cli: Add autocomplete functionality for flags for all CLI command [GH 3087]
```

Comment:
So we order the commands as follows:
1) core first
2) Then alphabetically grouped by category.
3) Within a category (all "cli" commands), we order by line-length.